src/components/ExportButton/ExportButton.js

```diff
@@ -60,7 +68,7 @@ function ExportButton({
         opacity: isLoading ? 0.3 : 1,
         ...style,
       }}
-      extraData={generateExtraData(layers)}
+      extraData={exportCopyright ? generateExtraData(layers) : null}
```

Comment:
it seems there is a second paramter exportNorthArrow. Maybe chnage the second  parameter  to an object :
generateExtraData(layers, { exportCopyright }

Reply:
Can we pass the scaleCanvas in the extradata props ?